modules/lb_repeat_schedules/config/install/core.entity_form_display.block_content.lb_repeat_schedules.default.yml

```diff
@@ -0,0 +1,62 @@
+uuid: 4ef1cbaa-d06f-4625-9cc8-738e20924ec5
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.lb_repeat_schedules
+    - field.field.block_content.lb_repeat_schedules.body
+    - field.field.block_content.lb_repeat_schedules.field_rs_block
+  module:
+    - paragraphs
+    - text
+id: block_content.lb_repeat_schedules.default
+targetEntityType: block_content
+bundle: lb_repeat_schedules
+mode: default
+content:
+  body:
+    type: text_textarea_with_summary
+    weight: 26
+    region: content
+    settings:
+      rows: 9
+      summary_rows: 3
+      placeholder: ''
+      show_summary: false
+    third_party_settings: {  }
+  field_rs_block:
+    type: paragraphs
```

Comment:
@oleh-rymar we're trying to get away from Paragraphs eventually, but if this allows you to save a significant amount of time then I think it's ok for the first iteration. How long do you think it would take to move the fields from the Paragraph into a block to prevent the need for the nesting?

Reply:
Hi @froboy, we discussed timing in the Jira ticket: https://imagex.atlassian.net/browse/YDAL0625-109 
Since we have a limited budget, I guess we will need to go with paragraphs as a Beta anyway.